sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -177,7 +177,11 @@ case class FileSourceScanExec(
 
   private lazy val needsUnsafeRowConversion: Boolean = {
     if (relation.fileFormat.isInstanceOf[ParquetSource]) {
-      SparkSession.getActiveSession.get.sessionState.conf.parquetVectorizedReaderEnabled
+      SparkSession.getActiveSession.orElse(SparkSession.getDefaultSession).map { session =>
+        session.sessionState.conf.parquetVectorizedReaderEnabled
+      }.getOrElse {
+        SQLConf.get.parquetVectorizedReaderEnabled
```

Comment:
@viirya, can't we just use `SQLConf.get.parquetVectorizedReaderEnabled` directly? 
Just to clarify, the behaviour here we want is:
- Use the conf in active Spark session
- If not, use default one.
?

Reply:
Yea, sounds good. Looks like we do that in most cases.

Reply:
+1, if active session is set, `SQLConf.get` will read conf from active session.

Reply:
Oh, I see. Removed active SparkSession getter and call `SQLConf.get.parquetVectorizedReaderEnabled` directly.